### DISCUSSION TITLE
SageTV and dvd rips - preferred methods?

### DISCUSSION THREAD

**Discussion by Sagetvmaster** [OpenSageTV/Community#65] [Capture Devices]
Anyone using DVDFab for ripping STANDARD def DVD's to a hard drive for SageTV to playback?
There are so many variants of DVDFab at their web site, I'm not sure which is the one for just simple DVD to disk ripping, anyone like to chime in here?  Or, whatever other methods you may use, mention that as well if you like.
I'd like to take all my DVD's to the thrift store, but would like them inside Sage first

**Comment by Ugadata** [comment]
AnyDVD HD from Redfox (Slysoft) is one option.  It has been a while since I used it but I think you have the option of ripping to an ISO file or folder format.  It does DVD but also does BluRay's.   I believe they have a trial.  This is just a straight rip and will keep the menu structure intact.  Various purchase options but a lifetime purchase is 110 EUR.  Just keep in mind, lifetime doesn't mean forever.

MakeMKV - Does both BluRay's and DVD's.  You can backup a BluRay disk (folder structure) or just create an MKV.  With DVD's you can only create an MKV.  Not sure if ripping to MKV will keep the menu system, I have only wanted the movie.  MakeMKV is free while in beta (it has been in beta for a very long time!!!!!).  There are fairly regular updates.

Both of these products have been around for quite some time.

**Reply by Sagetvmaster** [reply]
thanks
